lib/oeqa/selftest/cases/updater_qemux86_64.py

```diff
@@ -86,7 +89,7 @@ def setUpLocal(self):
         self.append_config('SOTA_CLIENT_PROV = " aktualizr-shared-prov "')
         self.append_config('IMAGE_FSTYPES_remove = "ostreepush garagesign garagecheck"')
         self.append_config('SOTA_HARDWARE_ID = "plain_reibekuchen_314"')
-        self.qemu, self.s = qemu_launch(machine='qemux86-64')
+        self.qemu, self.s = qemu_launch(machine='qemux86-64', uboot_enable=UBOOT_ENABLE)
```

Comment:
Maybe it can be a default param, so no need to specify it in each call.

Reply:
Actually the default parameter is ```yes``` which is ok for ostree cases, for the non-ostree, in order to make those tests reusable we have to explicitly set it to ```no```

Reply:
Do we need to specify these at all if the default is `yes`?